webpack.config.js

```diff
@@ -252,7 +252,7 @@ if ( isDevelopment ) {
 	webpackConfig.devtool = '#eval';
 } else {
 	webpackConfig.entry.build = path.join( __dirname, 'client', 'boot', 'app' );
-	webpackConfig.devtool = false;
+	webpackConfig.devtool = process.env.SOURCEMAP || false;
```

Comment:
I don't think this will do what we expect it to. You need to parse it to a boolean. 

I would want `SOURCEMAP=false npm run XYZ` to make `devtool=false` but since 'false' the string is cast to true the boolean, it won't work.

Reply:
on the other hand, `false` is the default in the absence of the flag. I wouldn't expect anyone to set `SOURCEMAP=false` but rather set it to a legitimate value

but, we _could_ easily wrap it in `JSON.parse( process.env.SOURCEMAP || false )` I think

Reply:
oh nevermind, i see now that you are planning to use the string value for the kind of source-map instead of just true/false. Carry on!